README.md

```diff
@@ -19,14 +19,24 @@ So, the all-new goreleaser was born.
 
 ## Usage
 
-Basically, you need to create a `goreleaser.yml` file in the root of your
+```
+go get github.com/goreleaser/releaser
```

Comment:
Yeah, I saw that and thought it might work. You're welcome to close this pull request and add the documentation yourself, I just prefer pull requests over issues :)